src/styles/main.scss

```diff
@@ -1,3 +1,21 @@
+@import "variables.scss";
+@import "calendar.scss";
+@import "page.scss";
+
```

Comment:
You do not use tags `h1` - `h6`, `p`, `ol` in your HTML. 
There is no need to write styles for things that are not used in the code.